public/app/features/alerting/unified/components/rule-editor/CloudEvaluationBehavior.tsx

```diff
@@ -28,7 +28,7 @@ export const CloudEvaluationBehavior = () => {
     <RuleEditorSection stepNo={3} title="Set evaluation behavior">
       <Field
         label="Pending period"
-        description="Period in which an alert rule can be in breach of the condition until the alert rule fires."
+        description='Period the threshold condition must be met to trigger the alert. Selecting "None" triggers the alert immediately once the condition is met.'
```

Comment:
I'm not sure what the difference is you're trying to make with the first two sentences here? second sentence is fine

Reply:
I tried using two sentences because the `Pending period` option may not be obvious at first glance.

The second sentence reiterates the behavior explaining how the `None` option works, which users may wonder about this particular case. 

By using these two sentences, I hope the option behavior is a bit more clear.  I don't have a strong opinion, so happy to change it to what you think is best.